services/brig/brig.integration.yaml

```diff
@@ -137,6 +137,9 @@ optSettings:
   setUserCookieThrottle:
     stdDev: 5
     retryAfter: 1
+  limitFailedLogins:
```

Comment:
sry, can u make it `limitFailedLogins` -> `setLimitFailedLogins` for consistency's sake? ☺️